tests/airgapped/training/tfjob-simple.yaml

```
@@ -0,0 +1,22 @@
+# Based on the upstream [TFJob example](https://github.com/kubeflow/training-operator/blob/964a6e836eedff11edfe79cc9f4e5b7c623cbe88/examples/tensorflow/image-classification/create-tfjob.ipynb)
+apiVersion: "kubeflow.org/v1"
+kind: TFJob
+metadata:
+  name: tfjob-simple
+spec:
+   tfReplicaSpecs:
+     Worker:
+       replicas: 2
+       restartPolicy: OnFailure
+       template:
+         metadata:
+           annotations:
+             sidecar.istio.io/inject: "false"  # disable istio sidecar injection to get tfjob running
+         spec:
+           containers:
+             - name: tensorflow
+               image: 172.17.0.2:5000/kubeflow-ci/tf-mnist-with-summaries:1.0
```

```suggestion
               image: 172.17.0.2:5000/kubeflow-ci/tf-mnist-with-summaries:1.0
```
Should `172.17.0.2:5000` be hardcoded?

---

the test scripts have this registry IP [hardcoded](https://github.com/canonical/bundle-kubeflow/blob/main/tests/airgapped/registry.sh#L21) to this value
I can add a comment here